src/main/java/dev/snowdrop/release/App.java

```diff
@@ -310,28 +310,28 @@ public void updateConfigForRelease(
         @CommandLine.Option(names = {"-glu", "--gluser"}, description = "Gitlab user name", required = true) String gluser,
         @CommandLine.Option(names = {"-glt", "--gltoken"}, description = "Gitlab API token", required = true) String gltoken,
         @CommandLine.Option(names = {"-r", "--release"}, description = "release", required = true) String release,
-        @CommandLine.Option(names = {"-pr", "--previous-release"},description = "Previous release in the <major>.<minor>.<fix> format (e.g. 2.4.3)",required = true) String previousRelease,
         @CommandLine.Option(names = {"-q", "--qualifier"}, description = "qualifier", required = true) String qualifier,
-        @CommandLine.Option(names = {"-m", "--milestone"}, description = "milestone", required = true) String milestone)
+        @CommandLine.Option(names = {"-m", "--milestone"}, description = "milestone", required = true) String milestone,
+        @CommandLine.Option(names = {"-t", "--test"}, description = "Create a test release ticket using the SB project for all requests") boolean test)
         throws Throwable {
         LOG.infof("release: %s; qualifier: %s; milestone: %s", release, qualifier, milestone);
         String[] releaseMMF = release.split("\\.");
         final String gitFullRef = String.format("%s/sb-%s.%s.x", gitRef, releaseMMF[0], releaseMMF[1]);
         Release releaseObj = factory.createFromGitRef(gitFullRef, false, true, release);
-
-        GitConfig config = GitConfig.gitlabConfig(release, gluser, gltoken, buildConfigForkRepoName, Optional.of(String.format("sb-%s.%s.x", releaseMMF[0], releaseMMF[1])),Optional.empty());
+        releaseObj.setMildTest(test);
+        GitConfig config = GitConfig.gitlabConfig(release, gluser, gltoken, buildConfigForkRepoName, Optional.of(String.format("sb-%s.%s.x", releaseMMF[0], releaseMMF[1])), Optional.empty());
         git.initRepository(config);
-
-        git.commitAndPush("chore: update " + release + " release issues' key [release-manager]", config, repo -> Stream.of(buildConfigUpdateService
-            .updateBuildConfig(repo, releaseObj, release, qualifier, milestone)));
-
+        git.commitAndPush("chore: update " + release + " release issues' key [release-manager]", config, repo -> {
+            var bcFile = buildConfigUpdateService.updateBuildConfig(repo, releaseObj, release, qualifier, milestone);
+            return releaseObj.isTestMode() ? Stream.empty() : Stream.of(bcFile);
```

Comment:
Can we move this logic inside the `updateBuildConfig` methode as we have for the other cases?

Reply:
done